package.json

```diff
@@ -37,6 +37,7 @@
     "packages/commonwell-sdk",
     "packages/commonwell-cert-runner",
     "packages/commonwell-jwt-maker",
+    "packages/cq-cert-runner",
```

Comment:
can we follow the pattern we have for commonwell and have "carequality" instead of "cq"?